test/functional/wallet_migration.py

```diff
@@ -396,6 +396,20 @@ def test_pk_coinbases(self):
 
         assert_equal(bals, wallet.getbalances())
 
+    def test_encrypted(self):
+        self.log.info("Test migration of an encrypted wallet")
+        wallet = self.create_legacy_wallet("encrypted")
+        default = self.nodes[0].get_wallet_rpc(self.default_wallet_name)
+
+        wallet.encryptwallet("pass")
+        addr = wallet.getnewaddress()
+        default.sendtoaddress(addr, 1)
+        self.generate(self.nodes[0], 1)
```

Comment:
These lines seem to be unrelated to the behavior we are testing for, and the test compiles and runs perfectly for the branch and fails for the master (as expected). So, could these lines be removed?

```suggestion
```

Reply:
Done